cypress/e2e/functional/researcher_permission_forms.spec.js

```diff
@@ -0,0 +1,113 @@
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
+
+        // TODO: fix clicking on archived rows, then uncomment these:
+        // pf.clickPermissionFormArchive(TEST_PERMISSION_FORM_NAME+'2');
+        // pf.clickPermissionFormArchive(TEST_PERMISSION_FORM_NAME+'3');
+        // pf.clickPermissionFormUnarchive(TEST_PERMISSION_FORM_NAME+'3');
+
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL);
+
+        cy.log('Test the Edit link, for an Archived, an Unarchived and a never-Archived Permision Form');
+        pf.clickPermissionFormEdit(TEST_PERMISSION_FORM_NAME+'2');
+        pf.verifyEditPermissionFormDialog(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL);
+        pf.clickCreateEditPermissionFormCancelButton();
+
+        pf.clickPermissionFormEdit(TEST_PERMISSION_FORM_NAME+'3');
+        pf.verifyEditPermissionFormDialog(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
+        pf.clickCreateEditPermissionFormCancelButton();
+
+        pf.clickPermissionFormEdit(TEST_PERMISSION_FORM_NAME+'1');
+        pf.verifyEditPermissionFormDialog(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL);
+        pf.clickCreateEditPermissionFormCancelButton();
+
+        // TODO: add more testing of the Edit Permission Form Dialog (without clicking the 'Cancel' button)
+
+        cy.log('Test the Delete link, for an Archived, an Unarchived and a never-Archived Permision Form');
+        pf.clickPermissionFormDelete(TEST_PERMISSION_FORM_NAME+'2');
+        pf.clickPermissionFormDelete(TEST_PERMISSION_FORM_NAME+'3');
+        pf.clickPermissionFormDelete(TEST_PERMISSION_FORM_NAME+'4');
+
+        cy.log('Move to the Manage Student Permissions tab, checking the Enabled & Disabled links before and after');
+        pf.verifyManageStudentPermissionsLinkEnabled();
+        pf.verifyCreateManageProjectPermissionFormsLinkEnabled(false);
+        pf.clickManageStudentPermissionsLink();
+        pf.verifyManageStudentPermissionsLinkEnabled(false);
+        pf.verifyCreateManageProjectPermissionFormsLinkEnabled();
+
+        // TODO: add tests of the Manage Student Permissions tab (see PT story #187863588)
+        cy.log('Test the Manage Student Permissions tab');
+
+        cy.log('Delete the remaining Permission Form(s) created by this test');
+        pf.clickCreateManageProjectPermissionFormsLink();
+        pf.clickPermissionFormDelete(TEST_PERMISSION_FORM_NAME+'1');
+
```

Comment:
@johnTcrawford Same with the delete, we are not ensuring whether the deleted permission forms are correctly deleted and it is not displayed.

Reply:
Good point. I also added this as a Task to PT #187863588.